config/locales/instructions/es.yml

```diff
@@ -11,8 +11,7 @@ es:
           adicional.
         modal:
           copy: Si no tiene su clave personal, verifique su identidad nuevamente.
-          heading: Para continuar, por favor regrese a la %{friendly_name} app e inicie
-            una sesión.
+          heading: NOT TRANSLATED YET
```

Comment:
I am suspicious of this one. It has an interpolation arg the others don't, and looks exactly like the translation for `go_back_to_mobile_app` later in the file.